### PR TITLE
TEST/UCT: Allow extending test_p2p_rma and test_p2p_mix.

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/4223/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
@evgeny-leksikov pls review

File: test/gtest/uct/test_p2p_mix.cc [hunk 1]
```diff
@@ -4,219 +4,204 @@
...
+void uct_p2p_mix_test::run(unsigned count) {
     if (m_avail_send_funcs.size() == 0) {
         UCS_TEST_SKIP_R("unsupported");
     }
 
     mapped_buffer sendbuf(m_send_size, 0, sender());
     mapped_buffer recvbuf(m_send_size, 0, receiver());
 
-    for (int i = 0; i < 10000; ++i) {
+    for (unsigned i = 0; i < 10000; ++i) {
```
Comment (evgeny-leksikov) [inline test/gtest/uct/test_p2p_mix.cc]: Shouldn't it be `i < count`?

Comment (yosefe) [inline test/gtest/uct/test_p2p_mix.cc]: right :)

**Pull request merged**